source/extensions/filters/network/thrift_proxy/config.cc

```diff
@@ -162,7 +162,7 @@ void ConfigImpl::processFilter(
   ENVOY_LOG(debug, "    thrift filter #{}", filter_factories_.size());
   ENVOY_LOG(debug, "      name: {}", string_name);
   ENVOY_LOG(debug, "    config: {}",
-            MessageUtil::getJsonStringFromMessage(proto_config.config(), true));
+            MessageUtil::getJsonStringFromMessage(proto_config.typed_config(), true));
```

Comment:
What does this look like interactively? Can you paste a Gist?

Reply:
Looks fine. See https://pastebin.com/KxzeMyMJ